assets/css/_tab_bar.scss

```diff
@@ -0,0 +1,40 @@
+.m-tab-bar {
+  background-color: $color-primary;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  // Required for pinning to the left side of the screen on the tablets
+  left: 0;
+  position: fixed;
+  top: inherit;
```

Comment:
In the Vehicle Properties Panel, you changed `inherit` to `0`. When would you use `0` and when would you use `inherit`?

Reply:
I think maybe `inherit` seemed required in at least some places to get things (e.g. fixed positioning) to work on the Samsung tablets for reasons I don't fully understand. It would be good to be sure to review all these changes on an actual tablet to be sure everything looks right, especially areas with fixed positioning.

Reply:
I changed `inherit` to `0` in the other files because when it was `inherit` the vehicle properties panel was positioned off of the page. I'm not fully sure if there even needs to be a value for `top` here -- it seems to work fine without it in chrome but i haven't tested without it on the tablets.